public/assets/dashboard/navBarIcon.js

```diff
@@ -106,7 +106,7 @@ export const XIcon = (props) => {
   );
 };
 
-export const Active = () => {
+export const ActiveTranction = () => {
```

Comment:
Lets try to use the right spellings:

```suggestion
export const ActiveTransaction = () => {
```